internal/helper/helper.go

```diff
@@ -126,3 +128,14 @@ func BoolPointer(b bool) *bool {
 func StringPointer(s string) *string {
 	return &s
 }
+
+func UpdateDataSourceSchema(d *schema.Schema) *schema.Schema {
+	dv := &schema.Schema{
```

Comment:
Can you check if this function already exists?

Reply:
yes we do have, but in my fork, I don't have feat/pde branch so that's why I have to add it again
while merging I'll take care for this.

Reply:
Thanks, please create an issue to track this.